components/config/src/config/search.rs

```diff
@@ -1,5 +1,19 @@
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub enum IndexFormat {
+    #[serde(rename = "json")]
+    Json,
+    #[serde(rename = "javascript")]
```

Comment:
you can tell serde to change casing for everything in one attribute: https://serde.rs/container-attrs.html#rename_all

Reply:
Done in d9acc9e59e3625d7c1646c2e31a4e3ee47b1d38e